uci_ssh_test.go

```diff
@@ -0,0 +1,67 @@
+package uci_test
+
+import (
+	"testing"
+
+	"github.com/digineo/go-uci"
+	"github.com/stretchr/testify/assert"
+)
+
+const USERNAME = "root"
+const PASSWORD = ""
+const HOST = "192.168.1.129"
+
+func TestWrapperConnect(t *testing.T) {
+	// ToDo: we should ship a test server!
```

Comment:
Indeed. For the tests however I'd be OK with running a real but isolated SSH server in parallel (i.e. Docker).